### PR TITLE
Charter Undici WG

### PR DESCRIPTION
**Pull request by Ethan-Arrowood**
Ref: https://github.com/nodejs/undici/issues/580

Not sure what else we'd like to add the responsibilities of this WG.

@ronag @mcollina @dnlup @delvedor feel free to comment with additional ideas.

### CODE REVIEW

**Comment by Ethan-Arrowood** [comment]
@mcollina should we (myself, ronag, and dnlup) plan on attending the next TSC meeting or is that not necessary?

File: WORKING_GROUPS.md [hunk 1]
```diff
@@ -443,6 +444,15 @@ Responsibilities include:
   * nodejs/ci-config-github-actions
   * nodejs/package-maintenance repository.
 
+### [Undici](https://github.com/nodejs/undici)
+
+The Undici Working Group is responsible for maintaing and developing the undici http client library.
```
Comment (richardlau) [inline WORKING_GROUPS.md:449]:
```suggestion
The Undici Working Group is responsible for maintaining and developing the undici http client library.
```

**Comment by mhdawson** [comment]
I'm not sure it needs to be a WG to get the work done, but not in opposition either.

**Comment by Ethan-Arrowood** [comment]
> I'm not sure it needs to be a WG to get the work done

I wasn't sure how much to expand in the responsibilities section, but I imagine this WG will be responsible for any all http client related pieces from the Undici module & maybe even node core. One such piece would be a fetch implementation I'm going to contribute to undici core in the coming weeks.

**Comment by mhdawson** [comment]
The responsibilities should cover everything that the WG would be "delegated" responsibility for.  That is the main difference between a team and a WG. A WG has been formally delegated the responsibility outlined in their charter.

**Comment by Ethan-Arrowood** [comment]
Just added another line. I encourage the other members of this WG to add things as well 👍

**Comment by Ethan-Arrowood** [comment]
I have no strong opinions either - just started the formal process initiated by https://github.com/nodejs/undici/issues/580

I'll let @mcollina take it from here 🙂

**Comment by mcollina** [comment]
The key reason Undici should be chartered as a working group is that Undici is really owned by the team maintaining it. The biggest form of independence that Node.js project provides is a chartered working group, so here it is.

We can call it "Next Generation HTTP clients for Node.js" - This will likely include some form of fetch() implementation in the future.

**Comment by mcollina** [comment]
Anyway, it seems there is more resistance than I anticipated. We should go with a team for now.

**Comment by MylesBorins** [comment]
To be clear I have no resistance, I just never assumed the TSC had any authority or oversight over the project to begin with.

If the team wants to be chartered and it helps them feel independent I'm +1

**Comment by Ethan-Arrowood** [comment]
FWIW: https://github.com/orgs/nodejs/teams/undici now exists

**Review by BridgeAR — APPROVED** [review]
If the team wants to be chartered, that's fine for me.

**Review by targos — APPROVED** [review]

**Review by BethGriggs — APPROVED** [review]

**Review by MylesBorins — APPROVED** [review]
LGTM

**Review by Trott — APPROVED** [review]

**Review by gabrielschulhof — APPROVED** [review]

**Review by mhdawson — APPROVED** [review]
LGTM

**Comment by mcollina** [comment]
Here it is: https://github.com/nodejs/undici/pull/626

**Review by gireeshpunathil — APPROVED** [review]

**Comment by mhdawson** [comment]
This looks like it should be ready to land. I'll do that next week unless somebody objections or lands it before I get to it.

**Pull request merged**